spec/version_spec.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'SitePrism::VERSION' do
+  subject { SitePrism::VERSION }
+
+  it { is_expected.to eq('2.11') }
```

Comment:
I'm somewhat skeptical of the value of this test. It basically says "Ruby has class constants and they work" which isn't a thing site_prism is really responsible for, plus this is a test that you'll definitely have to update every time you cut a release. Tests can sometimes be valuable just for the sake of documentation, but even on that score if I wanted to know the version I'd check the gemspec or, absent that, whatever file was defining the `VERSION` constant. It's not something I'd go looking for in the rspec tests.

I wouldn't block the PR over this - it's not a _lot_ of extra maintenance added when bumping version. Just not sure it's really necessary is all.